config/application.yml.example

```diff
@@ -188,6 +188,7 @@ test:
   logins_per_ip_limit: '2'
   logins_per_ip_period: '60'
   max_mail_events: '2'
+  mandrill_api_token: '123abc'
```

Comment:
This is an excellent reference for a PR requiring a corresponding application.yml config change:
- PR https://github.com/18F/identity-idp/issues/1488
- DevOps PR https://github.com/18F/identity-devops/pull/375